okhttp-graal/build.gradle

```diff
@@ -0,0 +1,28 @@
+jar {
+  manifest {
+    attributes('Automatic-Module-Name': 'okhttp3.graal')
+  }
+}
+
+dependencies {
+  api project(':okhttp')
+
+  compileOnly("org.graalvm.nativeimage:svm:20.2.0") {
+    // https://youtrack.jetbrains.com/issue/KT-29513
+    exclude group: 'org.graalvm.nativeimage'
```

Comment:
does this prevent us from using native images? or something else?

Reply:
kapt scans a compressed file that graal stores in the maven repository and fails.  This just says that we only want the annotations to compile against, we don't need the graal runtime.

Reply:
Should we be filing a Kotlin bug for that?

Reply:
It's linked in the bug above

Fixed for 1.4 in https://youtrack.jetbrains.com/issue/KT-41313, so maybe I can remove now.